frontend_app_gallery/src/component/ThumbnailPreview.jsx

```diff
@@ -1,19 +1,26 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import classnames from 'classnames'
+import { IMG_LOAD_STATE } from '../helper.js'
 
 class ThumbnailPreview extends React.Component {
   constructor (props) {
     super(props)
 
     this.state = {
-      imageLoaded: false
+      imageLoaded: IMG_LOAD_STATE.LOADING
     }
   }
 
-  onLoad () {
+  onLoad = () => {
```

Comment:
this naming is too generic, I know it existed before but now is a good time to rename it